iohub/convert.py

```diff
@@ -541,19 +542,25 @@ def _convert_ndtiff(self):
                         ndtiff_channel_idx,
                         z_idx,
                     )
-                    # row/well/fov/img/T/C/Z
+                    # T/C/Z
                     frame_key = "/".join(
-                        [zarr_arr.path]
-                        + [str(i) for i in (t_idx, c_idx, z_idx)]
+                        [str(i) for i in (t_idx, c_idx, z_idx)]
                     )
-                    all_ndtiff_metadata[frame_key] = image_metadata
-
-        logging.info("Writing ND-TIFF image plane metadata...")
-        with open(
-            os.path.join(self.output_dir, "image_plane_metadata.json"),
-            mode="x",
-        ) as metadata_file:
-            json.dump(all_ndtiff_metadata, metadata_file, indent=4)
+                    position_image_plane_metadata[frame_key] = image_metadata
+
+            logging.info("Writing ND-TIFF image plane metadata...")
+            # image plane metadata is save in
+            # output_dir/row/well/fov/img/image_plane_metadata.json,
+            # e.g. output_dir/A/1/FOV0/0/image_plane_metadata.json
+            with open(
+                os.path.join(
+                    self.output_dir, zarr_arr.path, "image_plane_metadata.json"
```

Comment:
I think `zarr_arr.path` is `/`-delimited regardless of the platform. This may not work on Windows.

Reply:
Tested that this doesn't cause problems. `os` and `pathlib` correctly handle `/` as a file separator on Windows.